### PR TITLE
details card

### PR DESCRIPTION
**Pull request by mmeiers**
@stulees @pcsandford @donnapep 
Here is the details card style and html. Let me know if you think anything needs to change. I still have some alignment styling to work out which I will do later.

### CODE REVIEW

**Comment by stulees** [comment]
@mmeiers :+1: if there's anything I run into that might need changing I'll let you know, thanks

File: src/css/style.css [hunk 1]
```diff
@@ -101,6 +101,71 @@ header:before {
...
+
+.details-name {
+  line-height: 8vw;
+  font-size: 6vw;
+  font-weight: 300;
+  margin:0;
+  padding-top:2vw;
+  padding-bottom:1vw;
+  margin-left:40px;
+  margin-right:40px;
```
Comment (pcsandford) [inline src/css/style.css:139]: could delete margin:0; and the left and right margins and just have margin:0 40px;

**Comment by pcsandford** [comment]
@mmeiers Looks good! I have commented on the margin as there are a few margin and padding entries that could be shortened, but as we will be tidying up after the live data comes in that is not a big deal right now. 

I don't think the alt tags need the hyphens or .png, they could just say "Well House Society Logo" for the screen reader to read them to someone in a more natural way (although this one may not ever use a screen reader it might be best practice going forwards?).

Do the spacings in px scale in the same way as the font and spacing using vh/vw? Again as this is a set size it probably won't matter, but I was just wondering in terms of reusing the code. 

Thanks!

**Comment by mmeiers** [comment]
@pcsandford 
Agree regarding alt and margin padding. I will be changing those.

Regarding spacing with px ya thats something we probably need to discuss in detail...

**Comment by mmeiers** [comment]
local stage: http://www.screencast.com/t/2mdJP5bHNI

**Comment by pcsandford** [comment]
@mmeiers cool, and the px stuff will definitely work itself out once the live data comes in and adjustments are made of needed then. Thanks!

**Pull request merged**